.build.sh

```diff
@@ -11,7 +11,7 @@ set -ue
 # - DEBUG
 
 # Source builder's functions library
-. /usr/local/share/cosmos-sdk/buildlib.sh
+. /usr/local/share/tendermint/buildlib.sh
```

Comment:
the base branch should be `release/v0.40.x`

Reply:
Yeah, forgot to to the right select.